java/client/src/main/java/glide/api/models/Script.java

```diff
@@ -18,13 +19,27 @@ public class Script implements AutoCloseable {
     /** Hash string representing the code. */
     @Getter private final String hash;
 
+    /** Indicatoin if script invocation output can return binary data. */
+    @Getter private final Boolean binarySafeOutput;
+
+    /**
+     * Wraps around creating a Script object from <code>code</code>.
+     *
+     * @param code To execute with a ScriptInvoke call.
+     */
+    public Script(String code, Boolean binarySafeOutput) {
```

Comment:
You can change this to use template instead of duplicating the code

```java
public Script(StringType code, boolean binarySafeOutput) {
      GlideString code = GlideString.of(code); // uses the variant that can accept any type
...
}
```